schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+ id integer,
```

Comment:
[OPTIONAL]
- It would be cool if you made the id column not null and auto increased for example ``id int NOT NULL IDENTITY(1,1)``
- You can also mark it as a primary key for example ``PRIMARY KEY (id)``